test/functional/interface_bitcoin_cli.py

```diff
@@ -90,6 +90,10 @@ def run_test(self):
         assert_raises_rpc_error(-8, "Parameter arg1 specified twice both as positional and named argument", self.nodes[0].cli.echo, 0, 1, arg1=1)
         assert_raises_rpc_error(-8, "Parameter arg1 specified twice both as positional and named argument", self.nodes[0].cli.echo, 0, None, 2, arg1=1)
 
+        self.log.info("Test that later cli named arguments values silently overwrite earlier ones")
+        assert_equal(self.nodes[0].cli("-named", "echo", "arg0=0", "arg1=1", "arg2=2", "arg1=3").send_cli(), ['0', '3', '2'])
+        assert_raises_rpc_error(-8, "Parameter args specified multiple times", self.nodes[0].cli("-named", "echo", "args=[0,1,2,3]", "4", "5", "6", ).send_cli)
```

Comment:
I'm not sure if it's worth changing, but seeing this test made me realize this error message could be confusing feedback to a user? They're not specifying `args` multiple times _explicitly_ - the fact that it's labelled `args` internally may not be immediately obvious? But also maybe it's a bit unnecessary to add special error handling for the `args` case.